gwt-uibinder-test/src/main/resources/org/gwtproject/uibinder/resources/UiBinder.xsd

```diff
@@ -0,0 +1,313 @@
+<?xml version="1.0"?>
```

Comment:
We shouldn't make it required to pull these files into each GWT module.    Rather, please update the GwtResourceEntityResolver line 73 to use `StandardLocation.CLASS_PATH` rather than the current `StandardLocation.CLASS_OUTPUT`    that'll eliminate the need to have these as it'll look at the classpath instead.